src/content/documentation/develop/web-ext-command-reference.md

```diff
@@ -339,6 +340,18 @@ web-ext run --start-url www.mozilla.com --start-url developer.mozilla.org
 
 Environment variable: `$WEB_EXT_START_URL`
 
+#### `--watch-ignored`
+A list of paths and globs patterns that should not be watched for changes. This is useful if you want to explicitly prevent web-ext from watching part of the extension directory tree, e.g. the node_modules folder.
+
+```shell
+web-ext run --watch-ignored path/to/file.js path/to/directory/to-be/ignored
+```
+
+::: note alert
+This option is most useful to linux users to avoid invoking `Error: ENOSPC: System limit for number of file watchers reached`. Visit [Issue #2022](https://github.com/mozilla/web-ext/issues/2022) for more context.
+:::
```

Comment:
As the --watch-ignored option description does not mention anything about `if this particular option is not used then an error may occur`. That's why I thought it was a good idea for whoever hits that error, will have an entry point as to what could be causing this issue instead of filing a new issue.
What do you think?

Reply:
> As the --watch-ignored option description does not mention anything about if this particular option is not used then an error may occur. That's why I thought it was a good idea for whoever hits that error, will have an entry point as to what could be causing this error instead of filing a new issue.
What do you think?

@ankushduacodes that's a pretty good point, maybe we can include it but without explicitly stating that it is a linux-only issue, e.g. something like

```
::: note alert

This option is useful to prevent issues when the number of watched files is higher than what the underlying OS feature allows.
As an example, on Linux a `Error: ENOSPC: System limit for number of file watchers reached` exception is raised if too many files are being watched (See [web-ext#2022](https://github.com/mozilla/web-ext/issues/2022)).
```

how that sounds?

Reply:
Sounds great!.. I will change that right away.